submissions/Nikita_Lemesiev/css_and_html_practice/style.css

```diff
@@ -129,21 +196,31 @@ header {
 	width: auto;
 	margin-bottom: 5px;
 }
+
+/*Кнопка More*/
```

Comment:
Such comments are redundant since if you name the class right, it tells about itself enough. Also, write comments in English. I'm writing this only here, but it is relevant to all your comments. It's fine to make comments to group some styles, like "Header styles" and then list styles for items that are relevant to the header somehow, but comments are rarely used for individual blocks, especially when only you control how to name classes.